content/docs/services/cloud-gov-service-account.md

```diff
@@ -4,7 +4,7 @@ menu:
     parent: services
 title: cloud.gov service account
 name: "cloud-gov-service-account"
-description: "Manage cloud.gov service accounts with access to your organization"
+description: "cloud.gov service accounts with access to your organization"
```

Comment:
Thinking that "cloud.gov service accounts with access to your organization" doesn't explain what the service account is _for_ - instead of talking about access to your organization or repeating the name, could this be something that defines what a service account is, such as "Specialized cloud.gov accounts for automated access by programs"?